docs/index.rst

```diff
@@ -1,108 +1,132 @@
-Piquasso
-********
+======================
+Piquasso documentation
+======================
+
+
+|
 
 
 Piquasso is an open source Python package, which allows you to simulate a photonic
-quantum computer. One could use Gaussian or Fock state corresponding to different
-representations to run simulations.
+quantum computer.
```

Comment:
Minor: this one is a very important sentence, it's like the "mission statement" of the project or at least the one-sentence summary of it. Some (not too over the top) extra formatting could be nice visually.